bin/verify_release_assets

```diff
@@ -0,0 +1,117 @@
+#!/bin/sh
+
+set -e
+
+. libstd-ecm.sh
+
+: "${REPOSITORY:=rancher/rke2}"
```

Comment:
What's the benefit of declaring these variables like this rather than 

```sh
REPOSITORY="ancher/rke2"
```

Reply:
It allows the user to override the value either through environment variable or flag. I could have done the same using the fallback `{:-}` shell expansion but is more verbose `REPOSITORY=${REPOSITORY:-rancher/rke2}`.

Also, it doesn't affect the script behavior (unless `set -u` is applied), but I feel more comfortable declaring all my locals at the beginning of the script for visibility and convenience.

Reply:
Gotcha. Let's make these simple declarations for consistency's sake with the rest of the scripts. We can set defaults here and the flags can override. 

Having options to for users to override values is nice but if we design these scripts such that they can only use flags and we dictate behavior, we'll have less bugs since there won't be as many moving parts to account for and further have to manage. Also, the other scripts only allow for variation through flags.